jobs/migrations/0011_alter_freelancer_role_level_and_more.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 4.0 on 2022-08-20 05:20

```

Comment:
Also wondering why this migration file got created if no underlying change to the models?